sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1764,6 +1794,11 @@ export const QueryMetricsConstants: {
     SchedulingMetricsText: string;
 };
 
+// @public (undocumented)
+export interface QueryOperationOptions {
+    ruCapPerOperation?: number;
```

Comment:
What does `ru` mean? Can we find a clearer name?

Reply:
It's a well known concept for request units in the case of Cosmos DB https://learn.microsoft.com/en-us/azure/cosmos-db/request-units